clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -266,41 +254,6 @@ public abstract class StoreIngestionTask implements Runnable, Closeable {
    */
   private final boolean suppressLiveUpdates;
 
-  /**
-   * This flag indicates whether the auto-compaction should be disabled for Samza Reprocessing Job.
-   * This feature has the assumption that most of the keys produced by Samza Reprocessing Job are
-   * unique, so we won't be too concerned about the storage amplification even though the storage size
-   * could be doubled during one-time manual compaction after receiving EOP.
-   * Since the compaction won't happen at the same time across all the storage partitions (different storage partition
-   * will hit EOP at different time and there are limit number of compaction threads), the storage
-   * amplification can be acceptable (need some experiment in prod).
-   *
-   * If this feature is proved to be useful, we could apply this optimization more widely to the
```

Comment:
Just want to learn more background on this. Did we find it not useful or we just find out we never enabled this on production?

Reply:
We didn't enable this in prod and the logic is too complex to maintain.
Currently, we rely on the compaction trigger to tune the compaction instead of this.